pyanaconda/iutil.py

```diff
@@ -1242,3 +1242,10 @@ def touch(file_path):
     # even when the path points to dirrectory
     if not os.path.exists(file_path):
         os.mknod(file_path)
+
+def firstNotNone(*args):
+    """Return the first argument that is not None."""
+    for arg in args:
+        if arg is not None:
+            return arg
```

Comment:
And if it is a function, it should be generic -- i.e. take any iterable as a single argument and return the first item that is not None. AFAICT there's no need to use ``*args`` machinery here.